wiki/Rules/Explicit_Content/de.md

```diff
@@ -0,0 +1,28 @@
+# Expliziter Inhalt
+
+![Screenshot einer Beatmap, die expliziten Inhalt enthält](img/explicit-tag-DE.jpg "Ein Beispiel einer Beatmap, die mit dem Tag 'explizit' markiert wurde.")
+
+Mapper können den Inhalt ihrer Beatmap als *explizit* zu deklarieren, indem sie die Checkbox `Explicit Content` ankreuzen, welche im [Genre- und Sprach-Popup-Menü](/wiki/Beatmap/Genre_and_language) auf dem Webeintrag der jeweiligen Beatmap zu finden ist. An diese Funktion sind einige Erwartungen und Regeln bezüglich ihrer Verwendung geknüpft.
+
+Ungeachtet der Art des Tons, stelle sicher, dass es den [generellen Regeln für Songinhalte](/wiki/Rules/Song_Content_Rules) entspricht.
+
+## Was zählt als expliziter Inhalt?
+
+Expliziter Inhalt meint primär den **Audioinhalt einer Beatmap**, meist in Bezug auf das Thema oder *intensive* Verwendung von expliziter Sprache. Das gilt **nicht** für visuelle Elemente einer Beatmap — diese **müssen** die [Empfehlungen für visuelle Inhalte](/wiki/Rules/Visual_Content_Considerations) einhalten.
+
+Allgemein gesprochen werden mit sehr wenigen Ausnahmen die meisten Formen von Musik für die Verwendung in Beatmaps akzeptiert, solange sie angemessen gekennzeichnet werden.
+
+## Was zählt als intensiver Verwendung von expliziter Sprache?
+
+Eine milde (oder einmalige) Verwendung von üblicher Profanität ist nicht ausreichend, damit eine Beatmap als explizit markiert werden kann. Beatmaps mit dieser Art von Sprache können gefahrlos bleiben, ohne dass sie als explizit markiert werden — solange die Sprache das ist, was man vernünftigerweise als etwas einstufen würde, das eine "elterliche Betreuung" (PG-13) voraussetzt. Falls Argumente darüber aufkommen, ob das für einen bestimmten Track relevant ist, dann nehme an, dass es das nicht ist.
+
+Der Gebrauch von Profanität muss fortwährend, signifikant, hetzerisch oder stark repetitiv sein, um als explizit zu zählen.
+
+Darüber hinaus sollten anschauliche und anhaltende Diskussionen von kontroversen Themen oder alles, was eine vernünftige Person als "heftig" bezeichnen würde, als explizit angesehen werden. Einige (aber nicht alle) Beispiele hier:
+
+- Symbolik, Auswirkung oder Konsequenzen von Selbstmord
+- Stark implizierte Gewalt
+- Darstellung/Diskussion von gewalttätigen Folgen in detaillierter Form
+- Extrem offensichtliche sexuelle Referenzen oder Anspielungen auf sexuelle Akte
+
+Eine gute Möglichkeit, zu erkennen, ob etwas als explizit markiert werden sollte oder nicht, ist zu schauen, ob irgendwelche großen Dienste für Audio-Streaming es als solches auf ihren eigenen Plattformen gelistet haben. [Spotify](https://www.spotify.com) ist ein hervorragender Ausgangspunkt.
```

Comment:
```suggestion
Eine gute Möglichkeit, zu erkennen, ob etwas als explizit markiert werden sollte oder nicht, ist zu schauen, ob irgendwelche großen Dienste für Audio-Streaming es als solches auf ihren eigenen Plattformen gelistet haben. [Spotify](https://www.spotify.com) ist ein hervorragender Einstiegspunkt.
```
maybe :thinking:

Reply:
I think `Ausgangspunkt` is more common. Deepl suggests both depending on if you add `excellent` to your input. The vague measurement of using Google returns about 400.000 results for `Einstiegspunkt` and around 13.000.000 for `Ausgangspunkt`. But both are fine of course 😄

Reply:
Yeah, I just thought `Ausgangspunkt` sounded weird as a translation for `starting point` even if it's correct lol, but let's keep it, because it's more common.